prow/plugins/reopen/reopen.go

```diff
@@ -74,9 +75,23 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 
 	if e.IsPR {
 		log.Info("Re-opening PR.")
-		return gc.ReopenPR(org, repo, number)
+		err := gc.ReopenPR(org, repo, number)
+		if err != nil {
+			if scbc, ok := err.(github.StateCannotBeChanged); ok {
+				resp := fmt.Sprintf("failed to re-open PR: %s", scbc)
```

Comment:
This and bellow should be `%v` instead of `%s`.

Reply:
done.